packages/thumbprint-react/components/ModalDefault/index.module.scss

```diff
@@ -204,6 +204,13 @@ $width-wide: 1400px;
     padding-top: 12px;
 }
 
+// The double selectors in this file ensure that consumers using the `className` prop can't easily
+// overwrite the styles that we want to enforce in the component.
+.modalContentFullBleed.modalContentFullBleed {
+    margin: 0 -32px !important;
```

Comment:
This is causing horizontal scrolling on mobile. 
![horizontalscroll](https://user-images.githubusercontent.com/916038/58919773-2b61b580-86e4-11e9-8557-32cdea98c24a.gif)

Haven't thought through the possible solutions much, but `overflow-x: hidden` may work.

Reply:
good catch. the padding actually needed to be responsive just like the padding for the main content element. still learning all the edge cases for this component.